### PR TITLE
Add Security Hub connector support to scanner

### PR DESCRIPTION
**Pull request by rdhariwal**
The goal of this PR is to take inspec findings map them to security hub findings and upload using a custom security hub integeration.

### CODE REVIEW

File: .github/workflows/docker-build.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: docker-build
 
 on:
   push:
-    branches: [main]
```
Comment (rdhariwal) [inline .github/workflows/docker-build.yml:5]: This code needs to be reverted once testing is complete

File: .github/workflows/docker-build.yml [hunk 2]
```diff
@@ -27,10 +31,12 @@ jobs:
       - name: Login to Amazon ECR
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
+      - name: Build go binary
+        run: go build main.go
       - name: Build the Docker image
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-        run: docker build . --file Dockerfile --tag $ECR_REGISTRY/$ECR_REPOSITORY:${TAG} --tag $ECR_REGISTRY/$ECR_REPOSITORY:latest --tag $ECR_REGISTRY/$ECR_REPOSITORY:${HASH}
+        run: docker build . --file Dockerfile --tag $ECR_REGISTRY/$ECR_REPOSITORY:${TAG} --tag $ECR_REGISTRY/$ECR_REPOSITORY:exp --tag $ECR_REGISTRY/$ECR_REPOSITORY:${HASH}
```
Comment (rdhariwal) [inline .github/workflows/docker-build.yml:39]: Revert the tag back to latest once testing is complete

File: scriptRunner.sh [hunk 3]
```diff
@@ -23,4 +23,11 @@ else
     echo "s3_bucket_path variable not found, skipping s3 results upload."
 fi
 
+echo "uploading findings into security hub"
+
+if [[ -n $PRODUCTARN ]]; then
```
Comment (rdhariwal) [inline scriptRunner.sh:28]: Consider changing this flag to a bool flag like `enable_securityhub` not necessary but might be worth considering.

**Review by chtakahashi — APPROVED** [review]
approved

**Pull request merged**